lib/model/folder_sendrecv.go

```diff
@@ -491,15 +491,26 @@ nextFile:
 			continue nextFile
 		}
 
+		// Verify there is some availability for the file before we start
+		// processing it
```

Comment:
Unrelated: Why are we even doing this so early? We might be able to pull it purely by copying. Seems like a check for the pull stage.

Reply:
This is also true